src/core.js

```diff
@@ -562,7 +562,7 @@ $.extend( $.validator, {
 			var lastActive = this.lastActive;
 			return lastActive && $.grep( this.errorList, function( n ) {
 				return n.element.name === lastActive.name;
-			} ).length === 1 && lastActive;
```

Comment:
> I still dont get where lastActive is changed between line 563 and 565 so, I dont get why it is checked against to be "truethy" again...

No, you get it wrong. The expression it's not clear and need a refractoring, something like this:
```js
		findLastActive: function() {
			var lastActive = this.lastActive,
				arr;

			if ( lastActive ) {
				arr = $.grep( this.errorList, function( n ) {
					return n.element.name === lastActive.name;
				} );

				return arr.length === 1 ? lastActive : false;
			}

			return false;
		},
```